### PR TITLE
Feature/fix timeseries basket

### PR DESCRIPTION
**Pull request by lindenmckenzie**
### What
Updated timeseries basket to closer styles to design system
Fixed broken javascript.

https://github.com/ONSdigital/dp-frontend-search-controller/assets/48557245/04a46ae7-ca69-4f86-81d6-782f801a35be

### How to review
Can be tested in partnership with this PR: https://github.com/ONSdigital/dp-design-system/pull/209

Switch on the aggregated data pages in dp-frontend-search-controller
Portforward dp-api-router to sandbox and test functionality for yourself.

Some things to test:

- add 1 item to basket using checkboxes
- 'select all items' add to checkboxes
- remove item from basket using checkboxes
- remove item from basket using remove button
- check for hidden inputs to download form appearing / disappearing
- check application memory of basket
- check no memory when remember selection is off

### Who can review
Frontend Dev.

### CODE REVIEW

**Comment by lindenmckenzie** [comment]
Requires update for design-system hash once approved.

File: assets/templates/partials/time-series-selection.tmpl [hunk 1]
```diff
@@ -2,41 +2,55 @@
...
-    <ul class="timeseries__list list--neutral margin-top-md--0">
-    </ul>
-    <div class="timeseries__download">
-        <p class="flush" id="timeseries__download-label">
-            {{ localise "DownloadSelectedTimeSeries" $lang 1 }}
-        </p>
+    <div class="timeseries__list--exit" aria-label="{{ localise "CloseTimeSeriesModal" $lang 1 }}" tabindex="0"><strong>✕</strong></div>
+    <div class="timeseries__inner timeseries__download">
+        <h3>{{ localise "TimeseriesBasketTitle" $lang 1 }}</h3>
+        <ul class="timeseries__list list--neutral margin-top-md--0">
```
[the remark this answers is not in the document]
Comment (lindenmckenzie) [reply assets/templates/partials/time-series-selection.tmpl]: Switched classes.

File: assets/templates/partials/time-series-selection.tmpl [hunk 2]
```diff
@@ -2,41 +2,55 @@
...
         <form id="xls-form" method="post" action="/export" class="inline print--hide">
             <button type="submit" title="Download XLS" aria-labelledby="timeseries__download-label"
-                class="btn btn--primary download-analytics">
-                {{ localise "DownloadFile" $lang 1 ".xls" }}
+                class="ons-btn ons-btn--secondary ons-btn--small ons-btn--download download-analytics">
+                <span class="ons-btn__inner">
+                    <svg class="ons-svg-icon ons-u-mr-xs" viewBox="0 0 12 12" xmlns="http://www.w3.org/2000/svg" focusable="false" fill="currentColor">
+                        <path d="M5.6 9a.48.48 0 0 0 .7 0l3-3.2a.48.48 0 0 0 0-.7C9.3 5 9.2 5 9 5H7.5V.5A.47.47 0 0 0 7 0H5a.47.47 0 0 0-.5.5V5H3a.47.47 0 0 0-.5.5.37.37 0 0 0 .1.3Z" />
+                        <path d="M11.5 9H11a.47.47 0 0 0-.5.5v1h-9v-1A.47.47 0 0 0 1 9H.5a.47.47 0 0 0-.5.5v2a.47.47 0 0 0 .5.5h11a.47.47 0 0 0 .5-.5v-2a.47.47 0 0 0-.5-.5Z" />
+                    </svg>
```
[the remark this answers is not in the document]
Comment (lindenmckenzie) [reply assets/templates/partials/time-series-selection.tmpl]: Called from dp-renderer - don't know why I didn't check there...I did for the button!

File: assets/templates/partials/time-series-selection.tmpl [hunk 3]
```diff
@@ -2,41 +2,55 @@
...
         <form id="csv-form" method="post" action="/export" class="inline print--hide">
             <button type="submit" title="Download CSV" aria-labelledby="timeseries__download-label"
-                class="btn btn--primary download-analytics">
-                {{ localise "DownloadFile" $lang 1 ".csv" }}
+                class="ons-btn ons-btn--secondary ons-btn--small ons-btn--download download-analytics">
+                <span class="ons-btn__inner">
+                    <svg class="ons-svg-icon ons-u-mr-xs" viewBox="0 0 12 12" xmlns="http://www.w3.org/2000/svg" focusable="false" fill="currentColor">
+                        <path d="M5.6 9a.48.48 0 0 0 .7 0l3-3.2a.48.48 0 0 0 0-.7C9.3 5 9.2 5 9 5H7.5V.5A.47.47 0 0 0 7 0H5a.47.47 0 0 0-.5.5V5H3a.47.47 0 0 0-.5.5.37.37 0 0 0 .1.3Z" />
+                        <path d="M11.5 9H11a.47.47 0 0 0-.5.5v1h-9v-1A.47.47 0 0 0 1 9H.5a.47.47 0 0 0-.5.5v2a.47.47 0 0 0 .5.5h11a.47.47 0 0 0 .5-.5v-2a.47.47 0 0 0-.5-.5Z" />
+                    </svg>
```
[the remark this answers is not in the document]
Comment (lindenmckenzie) [reply assets/templates/partials/time-series-selection.tmpl]: as above.

**Comment by lindenmckenzie** [comment]
> See comments on `dp-design-system`. I can't get this to work locally...

Now resolved - due to form nesting introduced by the one form approach.

**Review by mr-nick17 — DISMISSED** [review]
✅ - but you have merge conflicts

**Review by mr-nick17 — APPROVED** [review]

**Pull request merged**